packages/patternfly-3/patternfly-react/src/components/BreadcrumbSwitcher/components/BreadcrumbSearchInput.js

```diff
@@ -25,7 +25,7 @@ class BreadcrumbSearchInput extends React.Component {
           inputRef={input => {
             this.nameInput = input;
           }}
-          autocomplete="off"
```

Comment:
Since this is just a prop passed down to the DOM in https://github.com/nkbt/react-debounce-input/blob/master/src/Component.js, this is good.